src/config.js

```diff
@@ -19,6 +19,8 @@ module.exports = {
   VERSION: "V2.0.7",
   STATUS: "BETA",
 
+  GOOGLE_SEARCH_TOKEN: process.env.GOOGLE_SEARCH_TOKEN,
```

Comment:
Just realised you do not need to store it in the config to pass to the Layout file. You should be able to get it straight from the .env file. However, an update to the variable name is required as it needs to be prepended with 'GATSBY_' if you are going to take that approach.